ds_engine_nov_2021.py

```diff
@@ -4,38 +4,50 @@
 
 def prt_txt(text):
     for i in text:
-        print(f'{i}', end='', flush=True)
-        sleep(0.05)
-   
+        if i is "/":
+            print(f'{i}', end='', flush=True)
+        else:
+            print(f'{i}', end='', flush=True)
+            sleep(0.05)
+
+def take_in(accept):
```

Comment:
`accept` should be a list or a tuple (or a set) for stronger membership checking